openapi2kong/oas3_testfiles/19-request-validator-plugin-op-params-override-path-params.yaml

```
@@ -0,0 +1,36 @@
+# When the request-validator is added with path and operation parameters,
+# with both sharing same name and location, operation parameters override the path parameters.
+
+openapi: 3.0.2
+
+info:
+  title: Example
+  version: 1.0.0
+
+servers:
+  - url: http://backend.com/path
+
+x-kong-plugin-request-validator: {}
+
+paths:
+ /test/common-param/{common-param}:
+      parameters:
+        - in: path
+          name: common-param
+          schema:
+            type: integer
```

the parameters are both the same. Can we change the path level one to a `string` to verify that the operation level one overrides it?

```suggestion
            type: string
```